examples/puppeteer/app/src/lib.rs

```diff
@@ -11,13 +11,16 @@ use sails_rtl::calls::Call;
 use sails_rtl::gstd::calls::{GStdArgs, GStdRemoting};
 use sails_rtl::ActorId;
 
-pub struct Puppeteer {
-    puppet: Box<dyn ThisThatSvc<GStdRemoting, GStdArgs>>,
+pub struct Puppeteer<ThisThatClient> {
+    puppet: ThisThatClient,
 }
 
 #[gservice]
-impl Puppeteer {
-    pub const fn new(puppet: Box<dyn ThisThatSvc<GStdRemoting, GStdArgs>>) -> Self {
+impl<ThisThatClient> Puppeteer<ThisThatClient>
+where
+    ThisThatClient: ThisThatSvc<GStdRemoting, GStdArgs>,
```

Comment:
Nit: Using generic doesn't seem make sense here because you require uinsg GStdRemoting, i.e. I won't be able to use this service with some mock of ThisThatSvc which doesn't use remoting at all. Said that, it doesn't make much sense to abtract it